src/IntegrationUtils.ts

```diff
@@ -257,3 +310,19 @@ export async function isCamelKAvailable(): Promise<boolean> {
 		resolve(false);
 	});
 }
+
+function validateDependency(text : string) {
+	let inputEmpty : boolean = (text === undefined || text.trim().length === 0);
+	if (inputEmpty) {
+		return 'Dependency must be specified in the form of an Apache Camel component id or a Maven dependency in the form of group:artifact:version';
+	}
+	if (!inputEmpty && text.indexOf(':') > -1) {
+		let trifold : string[] = text.split(':');
+		// make sure we have three colon-delimited strings
+		return !(trifold && trifold.length === 3) ? 'Dependency must be specified in the form of a Maven dependency as group:artifact:version' : null;
+	} else if (!(!inputEmpty && text.indexOf('-') > -1)) {
```

Comment:
if I understand well, it requires a `-` in the component id.
I don't think that it is a requirements. For instance there are ahc, apns, amqp, file, timer, ... Or I'm looking to the wrong thing?

Reply:
its about artifactId I thought....like camel-ftp and so on

Reply:
Lars is right - if you look here - https://camel.apache.org/components/latest/ - and click through to ahc, apns, etc. - the dependency always seems to boil down to camel-component

Reply:
ok, then maybe worth precising i tin the message that i tis the "camel component artifact id" and not the "camel component id"